scripts/test-forge.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(cast chain --rpc-url="$ETH_RPC_URL")" == "ethlive" ]] || { echo "Please set a Mainnet ETH_RPC_URL"; exit 1; }
+
+for ARGUMENT in "$@"
+do
+    KEY=$(echo "$ARGUMENT" | cut -f1 -d=)
+    VALUE=$(echo "$ARGUMENT" | cut -f2 -d=)
+
+    case "$KEY" in
+            match)           MATCH="$VALUE" ;;
+            match-test)      MATCH_TEST="$VALUE" ;;
+            match-contract)  MATCH_CONTRACT="$VALUE" ;;
+            *)
+    esac
+done
+
+export DAPP_BUILD_OPTIMIZE=0   # forge turns on optimizer by default
```

Comment:
Shouldn't this script match the optimization parameters used elsewhere?

Reply:
Now getting this error:

![image](https://user-images.githubusercontent.com/2636663/201794284-ddeaefe9-82a6-4eec-a18e-538f55224f7f.png)

Reply:
Need to use either `FOUNDRY_OPTIMIZER_RUNS` or `DAPP_OPTIMIZER_RUNS`:

https://book.getfoundry.sh/reference/config/solidity-compiler?highlight=optimize#optimizer

Reply:
Fixed now.